src/main/java/com/booksaw/betterTeams/ConfigManager.java

```diff
@@ -154,6 +164,37 @@ public void updateFromDefaultSave(boolean log) {
 
 	}
 
+	private int updateVariables() {
+		if (!config.saveToString().contains("%s")) {
+			return 0;
+		}
+		int migratedKeys = 0;
```

Comment:
As this is only every treated as a boolean either by `migratedKeys !== 0` and `migratedKeys == 0` could this variable just be changed to a boolean as the number of migrated keys is never important

Reply:
Yeah it originally was a boolean, but I wanted to add a return value for verbosity like you have in your updateFileConfig method.